src/objects/config.ts

```diff
@@ -9,7 +9,7 @@ let config: RequiredGraphConfig | null = null
 const defaults: Omit<RequiredGraphConfig, 'runId' | 'fetch'> = {
   animationDuration: 500,
   disableAnimationsThreshold: 500,
-  disableEdgesThreshold: 100,
+  disableEdgesThreshold: 500,
```

Comment:
Not really a bug fix but we can easily render more edges and a pretty small flow I was looking at had more than 100 edges. So bumping this up to match the animation threshold.